README.md

```diff
@@ -1,29 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> One paragraph statement about the project.
+> Creating a protfolio to display my current skill level.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+Fully functional on Mobile.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML,CSS
+- No Frameworks
+- No Technologies
 
 ## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+images/demo.png
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
 
 To get a local copy up and running follow these simple example steps.
 
```

Comment:
# [OPTIONAL]
- [ ] This section should include instructions to other people on how they can set up this project locally, this way they can open your project on their computer and look at the code in their code editor. 👍🏻